client/extensions/wp-super-cache/cdn-tab.jsx

```diff
@@ -0,0 +1,179 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ExternalLink from 'components/external-link';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import FormToggle from 'components/forms/form-toggle/compact';
+import SectionHeader from 'components/section-header';
+import WrapSettingsForm from './wrap-settings-form';
+
+const CdnTab = ( {
+	fields: {
+		ossdl_cname,
+		ossdl_https,
+		ossdl_off_cdn_url,
+		ossdl_off_exclude,
+		ossdl_off_include_dirs,
+		ossdlcdn,
+	},
+	handleChange,
+	handleToggle,
+	siteUrl,
+	translate,
+} ) => {
+	return (
+		<div>
+			<SectionHeader label={ translate( 'CDN' ) }>
+				<Button
+					compact={ true }
+					primary={ true }
+					type="submit">
+						{ translate( 'Save Settings' ) }
+				</Button>
+			</SectionHeader>
+
+			<Card>
+				<form>
+					<FormFieldset>
+						<FormToggle
+							checked={ !! ossdlcdn }
+							onChange={ handleToggle( 'ossdlcdn' ) }>
+							<span>
+								{ translate( 'Enable CDN Support' ) }
+							</span>
+						</FormToggle>
+					</FormFieldset>
+
+					<FormFieldset>
+						<FormLabel htmlFor="ossdl_off_cdn_url">
+							{ translate( 'Off-site URL' ) }
+						</FormLabel>
+
+						<FormTextInput
+							id="ossdl_off_cdn_url"
+							onChange={ handleChange( 'ossdl_off_cdn_url' ) }
+							value={ ossdl_off_cdn_url || '' } />
+
+						<FormSettingExplanation>
+							{ translate(
+								'The new URL to be used in place of %(url)s for rewriting. No trailing / please.',
+								{
+									args: { url: siteUrl },
```

Comment:
In your example, this is showing `donnapep.wordpress.com` instead of `donnapep.wpsandbox.me` Seems like there might be an issue with this pulling the wrong url here?